packages/serverless-components/nextjs-component/src/component.ts

```diff
@@ -22,6 +22,7 @@ import type {
 type DeploymentResult = {
   appUrl: string;
   bucketName: string;
+  distributionId: unknown;
```

Comment:
Could this be made a string instead? 

```suggestion
  distributionId: string;
```

Reply:
done